rbui.gemspec

```diff
@@ -13,9 +13,9 @@ Gem::Specification.new do |s|
     "https://rubygems.org/gems/rbui"
   s.license = "MIT"
 
-  s.required_ruby_version = ">= 3.2"
+  s.required_ruby_version = ">= 3.3"
 
-  s.add_dependency "phlex", "~> 1.10"
```

Comment:
@SethHorsley, maybe it would be better to use `s.add_dependency "phlex", ">= 1.10"`. 

This way, we're not requiring everyone to use Phlex 2.0.

Reply:
good point!